EbandPlannerROS:
  #Common Parameters
  xy_goal_tolerance: 0.1
  yaw_goal_tolerance: 0.05
  rot_stopped_vel: 0.20
  trans_stopped_vel: 0.15


  #Trajectory Controller Parameters
  max_vel_lin: 0.5
  min_vel_lin: 0.15

  max_vel_th: 0.8 # choose slightly less than the base's capability
  min_vel_th: 0.20

  min_in_place_vel_th: 0.20
  in_place_trans_vel: 0.0

  Ctrl_Rate: 5.0

  max_acceleration: 1.0
  virtual_mass: 0.75

  max_translational_accel: 1.0
  max_rotational_acceleration: 1.5
  rotation_correction_threshold: 0.5
  differential_drive: false
